cmd/ff-proxy/main.go

```diff
@@ -308,25 +306,6 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Read replicas don't need to care about populating the repos with config
-	if !readReplica {
-		if err := conf.FetchAndPopulate(ctx, inventoryRepo, authRepo, flagRepo, segmentRepo); err != nil {
-			logger.Error("failed to populate repos with config", "err", err)
-			os.Exit(1)
-		}
-	}
-
-	// If the generateOfflineConfig flag is provided then we can just export the config and exit
-	if generateOfflineConfig {
-		exportService := export.NewService(logger, flagRepo, targetRepo, segmentRepo, authRepo, nil, configDir)
-		err = exportService.Persist(ctx)
-		if err != nil {
-			logger.Error("offline config export failed err: %s", err)
-			os.Exit(1)
-		}
-		os.Exit(0)
-	}
```

Comment:
I removed this GenerateOfflineConfig because we aren't concerned with ProxyV2 supporting this atm